src/pages/AppDetail.svelte

```diff
@@ -67,6 +72,8 @@
         <a href={`/apps/${params.app}/metrics/${metric.name}`}>{metric.name}</a>
         <i>{metric.description}</i>
       </li>
+    {:else}
+      <p>Your search didn't match any metric.</p>
```

Comment:
There's a bit of overlap between this and #129. We should probably account for the possibility that there are no metrics associated with an application (however rare that possibility may be in practice).

Reply:
Yes, you are right, it is showing the error message when there is no metrics associated. 
![image](https://user-images.githubusercontent.com/14244685/96645722-cd6f5580-134c-11eb-8efe-fbb264caad6e.png)

Should I wait till #129 gets marged and after that work on this PR @wlach?

Reply:
Yes that makes sense! I expect it will be merged soon, the code looks good just waiting on @DawoodShahat to fix a merge conflict.

Reply:
#129 has been merged